scripts/pre_push_hook.py

```diff
@@ -295,6 +295,16 @@ def _install_hook():
         print 'Copied file to .git/hooks directory'
 
 
+def does_diff_include_js_files(files_to_lint):
+    """Checks if frontend tests needs to be run."""
```

Comment:
This doc string seems to be wrong -- the function checks if the diff includes JS files, right?

Reply:
Done.